src/Raven.Server/Web/System/BuildVersionHandler.cs

```diff
@@ -49,7 +48,7 @@ private static byte[] GetVersionBuffer()
         public async Task Get()
         {
             HttpContext.Response.Headers.Add(Constants.Headers.ServerStartupTime, ServerStore.Server.Statistics.StartUpTime.GetDefaultRavenFormat(isUtc: true));
-            HttpContext.Response.Headers.Add(Constants.Headers.IsLinux, IsLinux);
+            HttpContext.Response.Headers.Add(Constants.Headers.IsLinux, IsLinux.ToString());
```

Comment:
This is the wrong place and method to do this. 
You want to decide what to show to the user for memory, put the string you want, on the server side, where you fetch the value from the server.
Otherwise, it will be hell to figure out what affects this.

Reply:
done